pkg/cache/application.go

```diff
@@ -232,6 +232,24 @@ func (app *Application) getTasks(state string) []*Task {
 	return taskList
 }
 
+func (app *Application) getNonTerminatedTaskAlias() []string {
+	var nonTerminatedTaskAlias []string
```

Comment:
looks like this can be simplified like the following:

```
var nonTerminatedTaskAlias []string
	for _, task := range app.taskMap {
		for _, states := range events.States().Task.Terminated {
			if states == task.GetTaskState() {
				continue
			}
		}
             nonTerminatedTaskAlias = append(nonTerminatedTaskAlias, task.alias)
	}
	return nonTerminatedTaskAlias
```

Reply:
Maybe we can add continue label `re`  ahead of the for of  app.taskMap and use `continue re ` to avoid append the terminated task.